lib/bolt/node/errors.rb

```diff
@@ -1,10 +1,10 @@
 module Bolt
   class Node
-    class BaseError < StandardError
+    class BaseError < Bolt::Error
       attr_reader :issue_code
 
       def initialize(message, issue_code)
-        super(message)
+        super(message, kind)
```

Comment:
I'd opt for passing `super(message, kind, nil, issue_code)` personally to make it clear that `@issue_code` is handled by the parent. Minor quibble.